### ISSUE TITLE
Does not run with eslint 2.3.0

### ISSUE THREAD

**Issue by gzurbach** [godaddy/javascript#17]
A bug was introduced in the latest version of `eslint` (2.3.0), preventing us from running the style guide.

```
Running eslint with options: -c godaddy-style/dist/.eslintrc server common test
Error: Cannot find module 'estraverse-fb'
```

See issue: https://github.com/eslint/eslint/issues/5476

I'm sure this issue will be fixed in a couple days, but I wonder whether it would be wise to lock the dependencies in the `package.json` (i.e. change `"eslint": "^2.1.0"` to `"eslint": "2.2.0"`). 

I tend to run into less issues when controlling which versions I'm using. Just a suggestion :)

**Comment by gshively11** [comment]
Hi @gzurbach.  Based on that eslint issue, it definitely seems like something is amiss, however, I'm unable to reproduce it locally, on either v2 or v3 of godaddy-style.  Can you provide any more details about this one?  Your package.json would be a good start.  I might have a dependency that is masking the problem.

**Comment by gzurbach** [comment]
@gshively11 You likely have eslint 2.1 or 2.2 in your local npm cache. Since the guide specifies `"eslint": "^2.1.0"` your cache has a match and doesn't get the latest version. Try to run `rm -rf node_modules && npm cache clear && npm i`, you should then be able to see the bug.

**Comment by hzoo** [comment]
babel-eslint isn't compatible with ESLint 2.3.x at the moment (I pushed a change that removes the specific error, but it still might be broken) so I would recommend pinning.

**Comment by gshively11** [comment]
@gzurbach I believe I did that, but I don't recall for sure.  I'm a bit swamped at the moment, will investigate this within a few days.  Apologies for the delay.

**Comment by indexzero** [comment]
Fixed in #40.
